QuestionAnswering/QA_ICL/backend/triton_inference_server/README.md

```diff
@@ -1,27 +1,32 @@
 # Self-hosting ML models for QA applications
 
-This directory contains resources to serve ML models with Triton Inference Server. The following models are included:
-
-- mpnet: text embedding model
+This directory contains the configuration to serve the Sentence-BERT model for generating text embedding.
```

Comment:
Maybe include which step of the workflow do we do text embedding

Reply:
A description has been added to provide the role of the server.